manifests/params.pp

```diff
@@ -25,6 +25,12 @@
       $sentinel_log_file         = '/var/log/redis/redis-sentinel.log'
       $sentinel_working_dir      = '/var/lib/redis'
 
+      $apt_location              = 'https://packages.redis.io/deb/'
+      $apt_repos                 = 'main'
+      $apt_release               = $facts.dig('os', 'distro', 'codename')
```

Comment:
This is the same value as the default in `apt::source` so I'd keep it `undef` by default rather than trying to duplicate it.